scripts/platoon-launcher

```diff
@@ -38,28 +38,37 @@ if __name__ == '__main__':
     os.makedirs(logs_folder)
 
     print "### Launching experiment: {}".format(args.experiment_name)
-    process_list = {}
+    process_map = {}
 
-    process_list['Controller'] = launch_process(logs_folder, args.experiment_name, args.controller_args, "cpu", "controller")
+    p = launch_process(logs_folder, args.experiment_name, args.controller_args, "cpu", "controller")
+    process_map[p.pid] = ('Controller', p)
```

Comment:
It's very relevant to this PR but if you insist, sure let make a separate issue.